packages/client/src/constants/maps.js

```diff
@@ -49,3 +70,27 @@ export const SAFE_TYPES_TO_META = {
     displayText: "Other",
   },
 };
+
+/** 
+ * TODO: Remove once API for fetching the name of the token is implemented
+ * For now every time the fungible token is added to flow.json, the token name should be added here 
+ * (in case it is different from the contract name) 
+*/
+export const CONTRACT_NAME_TO_COIN_TYPE = (contractName) => {
```

Comment:
here since it's a function, maybe `getCoinTypeFromContractName` make more sense?

Reply:
Changed it to a map to keep it as a constant
```javascript
export const CONTRACT_NAME_TO_COIN_TYPE = {
  "FlowToken": "FLOW",
  "FUSD": "FUSD",
  "FiatToken": "USDC",
  "BloctoToken": "BLT",
  "StarlyToken": "STARLY",
  "REVV": "REVV"
}
```